Improve text in starter guide

nop33 opened the pull request:
1. Improved readability of the _Requirements_ section by adding a footnote, fixing indentation, fixing typos, and updating the text
2. Improved readability of the _Start your node_ section by fixing indentation, creating a code block for easy copy+paste of command, and adding emoji for fun

Visual comparison:

| Before  | After |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/1579899/141792794-e3276f3f-57fd-4e2a-a854-c20c64a82ad8.png) | ![image](https://user-images.githubusercontent.com/1579899/141792840-295466b3-db72-47d4-8e97-66003dacb5ff.png)  |

polarker commented on src/Starter-Guide---How-to-Launch-your-node.md:
```diff
@@ -1,23 +1,20 @@
- Make sure to show the hidden folders:
- - [on Windows](https://support.microsoft.com/en-us/windows/view-hidden-files-and-folders-in-windows-97fbc472-c603-9d90-91d0-1166d1d9f4b5)
- - [on MacOS](https://www.pcmag.com/how-to/how-to-access-your-macs-hidden-files)
+3. In your home folder[^1]:
```
Very sweet footnote ❤️

tdroxler replied:
nice, I never did a footnote in md

tdroxler commented on src/Starter-Guide---How-to-Launch-your-node.md:
```diff
@@ -1,23 +1,20 @@
-      - In the variable name type `HOME`
-      - In the variable value: `C:\Users\<your-user-name>`
+   - To do this you can follow the `Windows` section of [this guide](https://java.com/en/download/help/path.html).
+   - Add the following two values in your user's variables:
```
can you fix the `two`? now there's only one variable (we use to have 2)

nop33 replied:
done

polarker reviewed (approved):
LGTM

tdroxler reviewed (approved):
one comment, otherwise LGTM

nop33 commented:
@polarker @tdroxler thanks for your feedback!

I've added the suggestions and moreover I:
1. added a title to the document and changed the hierarchy of the titles of the document's sections from H1 to H2
2. highlighted the explorer link by adding it inside a blockquote

Latest preview:

![image](https://user-images.githubusercontent.com/1579899/141794512-98a93968-ff4d-4fa2-bc80-fabe26dda16f.png)

polarker commented:
Feel free to merge now, thanks !
